src/main/resources/db/devonly/R__dev_data.sql

```diff
@@ -75,5 +75,23 @@ FROM acquirer_subscription
 WHERE event_type = 'DEATH_NOTIFICATION'
   AND oauth_client_id IN ('dwp-event-receiver', 'hmrc-client');
 
+
+INSERT INTO supplier (name) VALUES ('HMPPS');
```

Comment:
Could we also delete the things we add :)

Reply:
Sorry not sure what you mean!

Reply:
This is a repeatable migration, so it should be idempotent. Something like `DELETE FROM supplier WHERE NAME = 'HMPSS';` before this.

Reply:
👍

Reply:
Done